django/contrib/gis/db/backends/base/operations.py

```diff
@@ -48,6 +48,16 @@ class BaseSpatialOperations(object):
     # Aggregates
     disallowed_aggregates = ()
 
+    # Blacklist of known unsupported functions of the backend
+    unsupported_functions = (
+        'Area', 'AsGeoHash', 'AsGeoJSON', 'AsGML', 'AsKML', 'AsSVG',
+        'BoundingCircle', 'Centroid', 'Difference', 'Distance', 'Envelope',
+        'ForceRHR', 'Intersection', 'Length', 'MemSize', 'NumGeometries',
+        'NumPoints', 'Perimeter', 'PointOnSurface', 'Reverse', 'Scale',
+        'SnapToGrid', 'SymDifference', 'Transform', 'Translate',
+        'Union',
+    )
+
```

Comment:
What about making `unsupported_functions` a `set` instead? It looks like it is mostly used for existence checks anyway.